app/operator_cmds/commands.py

```diff
@@ -0,0 +1,113 @@
+import os
+
+import click
+from ocp_resources.namespace import Namespace
+from ocp_resources.operator import Operator
+from ocp_resources.operator_group import OperatorGroup
+from ocp_resources.subscription import Subscription
+from ocp_utilities.infra import get_client
+
+
+def _client(ctx):
+    return get_client(config_file=ctx.obj["kubeconfig"])
+
+
+@click.group()
+@click.option("--debug", help="Enable debug logs", is_flag=True)
+@click.option(
+    "--kubeconfig",
+    help="Path to kubeconfig file",
+    required=True,
+    default=os.environ.get("KUBECONFIG"),
+    type=click.Path(exists=True),
+    show_default=True,
+)
+@click.option("-n", "--name", help="Operator name to install/uninstall", required=True)
+@click.pass_context
+def operator(ctx, kubeconfig, name, debug):
+    """
+    Command line to Install/Uninstall Operator on OCP cluster.
+    """
+    ctx.ensure_object(dict)
+    ctx.obj["name"] = name
+    ctx.obj["kubeconfig"] = kubeconfig
+    if debug:
+        os.environ["OCM_PYTHON_WRAPPER_LOG_LEVEL"] = "DEBUG"
+        os.environ["OPENSHIFT_PYTHON_WRAPPER_LOG_LEVEL"] = "DEBUG"
+
+
+@operator.command()
+@click.option(
+    "-c",
+    "--channel",
+    help="Operator channel to install from",
+    default="stable",
+    show_default=True,
+)
+@click.option(
+    "--target-namespace",
+    help="target namespace for the operator. multiple parameters are allowed",
+    multiple=True,
+)
+@click.pass_context
+def install(ctx, channel, target_namespace):
+    """Install cluster Operator."""
+    client = _client(ctx=ctx)
+    name = ctx.obj["name"]
+
+    if target_namespace:
+        for namespace in target_namespace:
+            ns = Namespace(client=client, name=namespace)
+            if ns.exists:
+                continue
+
+            ns.deploy(wait=True)
+
+        OperatorGroup(
+            client=client,
+            name=name,
+            namespace=name,
+            target_namespaces=list(target_namespace),
+        ).deploy(wait=True)
+    else:
+        ns = Namespace(client=client, name=name)
+        if not ns.exists:
+            ns.deploy(wait=True)
+
+    Subscription(
+        client=client,
+        name=name,
+        namespace=name,
+        channel=channel,
+        source="redhat-operators",
```

Comment:
This would limit the tool to be used against production version only.

Reply:
I know, this is what we need for now.